src/core/services/LabService.ts

```diff
@@ -25,7 +25,7 @@ import y3CrvBackZapperAbi from './contracts/y3CrvBackZapper.json';
 import yvBoostAbi from './contracts/yvBoost.json';
 import pickleJarAbi from './contracts/pickleJar.json';
 import pickleGaugeAbi from './contracts/pickleGauge.json';
-
+import { BackscracherApyComposite } from '@src/../../yearn-sdk/dist';
```

Comment:
@gotzenx @xgambitox  is there a better import for this?

Reply:
oh this got stuck bc of a yarn link, will update